lib/inky/inky.ex

```diff
@@ -0,0 +1,625 @@
+defmodule Inky do
+  @moduledoc """
+  Documentation for Inky.
+  """
+
+  @doc """
+  Hello world.
+
+  ## Examples
+
+      iex> alias Inky
+      iex> state = Inky.setup(nil, :phat, :red)
```

Comment:
Technically you can avoid passing `nil` here since you have a default value set in the function header. 🙂

Reply:
Fair :)